conf/ocsci/downstream_config.yaml

```diff
@@ -11,3 +11,6 @@ ENV_DATA:
   # rook_csi_snapshotter_image:
   # rook_csi_attacher_image:
   cluster_namespace: 'rook-ceph'
+REPORTING:
+  # Upstream: 'US' or Downstream: 'DS'
+  us_ds: 'DS'
```

Comment:
I'm not sure this is a good idea. Maybe you can add into the comment that this is just for test run id format used in jenkins, and it should not be touched or used in any other way.

Reply:
It is in `REPORTING` section, which might be slightly indicating, that it is only for reporting... but I'll add it to the comment too.